sylvia/src/lib.rs

```diff
@@ -27,3 +27,5 @@ pub use {
     cosmwasm_schema as cw_schema, cosmwasm_std as cw_std, schemars, serde,
     serde_cw_value as serde_value, serde_json_wasm as serde_json,
 };
+#[cfg(feature = "orch")]
+pub use cw_orch;
```

Comment:
Conditionally export cw-orch here so that those lines like `#sylvia:: cw_orch::QueryFns` compile